sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -635,3 +635,21 @@ UPDATE `insurance_data` SET `date` = NULL WHERE `date` = '0000-00-00';
 #EndIf
 SET sql_mode = @currentSQLMode;
 
+#IfNotColumnType api_token token_api varchar(4)
+ALTER TABLE `api_token` MODIFY `token_api` varchar(4);
+#EndIf
+
+-- Note removing all data from api_token table in case legacy stuff gets in way
+--  and to ensure will not break when add the unique index below
+#IfNotColumnType api_token token varchar(40)
+TRUNCATE TABLE api_token;
```

Comment:
note I am emptying out the api_token table (figured not a big deal since just tokens here that expire in an hour) here to ensure the unique index works below